src/libsystemd/sd-device/sd-device.c

```diff
@@ -2327,6 +2323,16 @@ _public_ int sd_device_open(sd_device *device, int flags) {
         if (FLAGS_SET(flags, O_PATH))
                 return TAKE_FD(fd);
 
+        r = sd_device_get_property_value(device, "ID_IGNORE_DISKSEQ", &val);
+        if (r < 0 && r != -ENOENT)
+                return r;
+
+        if (!val || parse_boolean(val) <= 0) {
```

Comment:
for later: maybe take inspiration from getenv_bool(), and add a helper `device_get_property_value_bool()` that does this in one step?